tests/unit/utils/test_fs.py

```diff
@@ -216,6 +216,23 @@ def test_path_isin_with_absolute_path():
     assert path_isin(child, parent)
 
 
+def test_path_isin_case_sensitive():
+    child = os.path.join("path", "to", "folder")
+    parent = os.path.join("PATH", "TO")
+
+    assert path_isin(child, parent) == (os.name == "nt")
+
+
+def test_contains_symlink_case_sensitive():
+    child = os.path.join("path", "to", "folder")
+    parent = os.path.join("PATH", "TO")
+    if os.name == "nt":
+        assert contains_symlink_up_to(child, parent) is False
+    else:
+        with pytest.raises(BasePathNotInCheckedPathException):
+            contains_symlink_up_to(child, parent)
```

Comment:
@pmrowla, could you please split this test - one for windows, and one for other platforms?